### ISSUE TITLE
Crash on 32-bit iOS simulator on Mac with 16Gb+ of RAM

### ISSUE THREAD

**Issue by RuiAAPeres** [kean/Nuke#13]
With the following command:

`./xctool -workspace <workspace> -scheme <scheme> -sdk iphonesimulator test`

I am getting this:

```
Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   Nuke.Nuke                     	0x004126ac _TFC4Nuke16ImageMemoryCacheCfMS0_FT_S0_ + 588
1   Nuke.Nuke                     	0x003e557a _TIFV4Nuke25ImageManagerConfigurationcFMS0_FT10dataLoaderPS_16ImageDataLoading_7decoderPS_13ImageDecoding_5cacheGSqPS_18ImageMemoryCaching___S0_A1_ + 42
2   Nuke.Nuke                     	0x003f1bd4 globalinit_33_26AEE105B8E103C743A4428E83DD33A7_func0 + 100
3   libdispatch.dylib             	0x032656fd _dispatch_client_callout + 14
4   libdispatch.dylib             	0x032514d5 dispatch_once_f + 83
5   libswiftCore.dylib            	0x02e26fda swift_once + 42
6   Nuke.Nuke                     	0x003f1c64 _TZFC4Nuke12ImageManager6sharedfMS0_FT_PS_13ImageManaging_ + 68
7   Nuke.Nuke                     	0x0040c27d _TF4Nuke21startPreheatingImagesFGSaVS_12ImageRequest_T_ + 29
```

Inside Xcode I was able to reproduce the crash very few times, while on Xctool always crashes. On Xcode the breakpoint hit here: https://github.com/kean/Nuke/blob/44e5e3b39757ab00b8d34c9b29798b931c1bd995/Nuke/Source/Core/ImageMemoryCache.swift#L65

**Comment by RuiAAPeres** [comment]
With the following works:

```
./xctool -workspace ~/Documents/MailOnline/Money/Money.xcworkspace -scheme Money -sdk iphonesimulator -destination name="iPhone 6" test  
```

It seems to me the issue is related to 32bits devices (iPhone5 also crashes, but no the 5s)

**Comment by RuiAAPeres** [comment]
A bit more digging revealed the following:

```
fatal error: floating point value can not be converted to Int because it is greater than Int.max
```

**Comment by kean** [comment]
Fixed by #14
